training/src/main/scala/com/airbnb/aerosolve/training/AdditiveModelTrainer.scala

```diff
@@ -304,7 +318,7 @@ object AdditiveModelTrainer {
     val expCorr = scala.math.exp(corr)
     val loss = scala.math.log(1.0 + 1.0 / expCorr)
     val grad = -label / (1.0 + expCorr)
-    val gradWithLearningRate = grad.toFloat * params.learningRate.toFloat * classWeights(label.toInt)
+    val gradWithLearningRate = grad.toFloat * getLearningRate(label, params) * classWeights(label.toInt)
```

Comment:
`classWeights` can be used to adjust positive, negative sample importance or learning rate